scripts/core/exp-command-buffer.yml

```diff
@@ -704,3 +704,36 @@ returns:
         - "If event objects in phEventWaitList are not valid events."
     - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
+--- #--------------------------------------------------------------------------
+type: function
+desc: "Get profiling information for the sync point execution associated with an event object"
+class: $xSyncPoint
+name: GetProfilingInfoExp
+ordinal: "0"
+params:
+    - type: $x_event_handle_t
+      name: hEvent
+      desc: "[in] handle of the event object"
+    - type: $x_exp_command_buffer_sync_point_t
+      name: SyncPoint
+      desc: "[in] Sync point referencing the node (i.e. command) from which we want to get profile information"
+    - type: $x_profiling_info_t
+      name: propName
+      desc: "[in] the name of the profiling property to query"
+    - type: size_t
+      name: propSize
+      desc: "[in] size in bytes of the profiling property value"
+    - type: void*
+      name: pPropValue
+      desc: "[out][optional][typename(propName, propSize)] value of the profiling property"
+    - type: size_t*
+      name: pPropSizeRet
+      desc: "[out][optional] pointer to the actual size in bytes returned in propValue"
+returns:
```

Comment:
Could have an error ` $X_RESULT_ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_EXP` if `syncPoint` doesn't correspond to a graph node in event.

Reply:
This error has been added.

Reply:
thanks, could have a sub-bullet point describing when this error is returned